src/main/java/teammates/sqllogic/api/Logic.java

```diff
@@ -202,6 +204,13 @@ public Course getCourse(String courseId) {
         return coursesLogic.getCourse(courseId);
     }
 
+    /**
+     * Gets a section from a course by section name.
+     */
+    public Section getSection(String courseId, String section) {
+        return usersLogic.getSectionOrCreate(courseId, section);
```

Comment:
Should this be calling `.getSection` instead? Looks like a duplicate of `.getSectionOrCreate` below otherwise

Reply:
Will also add in a separate getSection method in the usersLogic level to resolve this